dev/ProgressBar/ProgressBar.cpp

```diff
@@ -21,6 +21,7 @@ ProgressBar::ProgressBar()
     RegisterPropertyChangedCallback(winrt::RangeBase::ValueProperty(), { this, &ProgressBar::OnRangeBasePropertyChanged });
     RegisterPropertyChangedCallback(winrt::RangeBase::MinimumProperty(), { this, &ProgressBar::OnRangeBasePropertyChanged });
     RegisterPropertyChangedCallback(winrt::RangeBase::MaximumProperty(), { this, &ProgressBar::OnRangeBasePropertyChanged });
+    RegisterPropertyChangedCallback(winrt::Control::PaddingProperty(), { this, &ProgressBar::OnRangeBasePropertyChanged });
```

Comment:
>OnRangeBasePropertyChanged [](start = 93, length = 26)

Maybe we should rename this handler, something like OnIndicatorWidthParameterChnaged, since this isn't a range base property which is using this handler.

Reply:
@carldebilly do you think you'd be able to make this update?